canonical-web-and-design/ubuntu.com
# Logout clears sessions not logs out of SSO

## Done
Logout clears sessions not logs out of SSO

## QA
- Go to /advantage
- Login
- Log out and see you land logged out on advantage
- Go to /core/build
- Login 
- Log out and see you return to /core/build logged out
- Login again and then change the URL to /logout
- See you land on the homepage

## Issue / Card
Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/9366
